src/source/tile.js

```diff
@@ -166,6 +167,12 @@ class Tile {
             }
         }
 
+        this.additionalRadius = 0;
+        for (const id in this.buckets) {
+            const bucket = this.buckets[id];
+            this.additionalRadius = Math.max(this.additionalRadius, painter.style.getLayer(bucket.layerIds[0]).queryRadius(bucket));
```

Comment:
Since we're storing the `additionalRadius` value here, can't we re-use the value in `Tile#queryRenderedFeature` instead of re-calculating? It's probably not that expensive to calculate, but it still seems like a good code simplification.